File: C/Colleen.c
#include <stdio.h>
/*
Goodbye World!
*/
void Bunny()
{
    char *str="#include <stdio.h>%c/*%cGoodbye World!%c*/%cvoid Bunny()%c{%c    char *str=%c%s%c;%c    printf(str, 10, 10, 10, 10, 10 , 10, 34, str, 34, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10);%c}%c%cint main()%c{%c    /*%c    Hop%c    */%c    Bunny();%c    return (0);%c}";
    printf(str, 10, 10, 10, 10, 10 , 10, 34, str, 34, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10);
}

int main()
{
    /*
    Hop
    */
    Bunny();
    return (0);
}

File: Python/Colleen.py
#Goodbye World!
def bunny():
    str = "#Goodbye World!{n}def bunny():{n}    str = {q}{s}{q}{n}    print(str.format(s=str, n=chr(10), q='{q}')){n}def main():{n}    #Hop{n}    bunny(){n}if __name__ == {q}__main__{q}:{n}    main()"
    print(str.format(s=str, n=chr(10), q='"'))
def main():
    #Hop
    bunny()
if __name__ == "__main__":
    main()
